lib/ui/window/platform_configuration.cc

```diff
@@ -35,7 +32,15 @@ PlatformConfigurationClient::~PlatformConfigurationClient() {}
 
 PlatformConfiguration::PlatformConfiguration(
     PlatformConfigurationClient* client)
-    : client_(client) {}
+    : client_(client) {
+  if (client_->ImplicitViewEnabled()) {
+    // Add PlatformConfiguration's window now, but don't add dart:ui's View
+    // here. The dart:ui needs another way to add the implicit view
+    // synchronously so that the view is available before the main function.
+    // See _implicitViewEnabled in natives.dart.
```

Comment:
```suggestion
    // See _implicitViewId in natives.dart.
```

Reply:
Done. I've also searched through the repo to make sure there are no other cases.